contracts/dtoken/tests/userflows/userflow.e2e.feature

```diff
@@ -0,0 +1,25 @@
+Feature: User e2e flow
+
+
+Background:
+	Given The user A
```

Comment:
@pahomas let's stick to default names (alice, bob, carol etc) because `User A`  reads as `a` article